src/Widgets/ConnectivityInfoBar.vala

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2011-2020 elementary, Inc. (https://elementary.io)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Authored by: Fabio Zaramella
+ */
+
+public class Calendar.Widgets.ConnectivityInfoBar : Gtk.InfoBar {
+    private GLib.NetworkMonitor network_monitor;
+    private Gtk.Label info_label;
```

Comment:
Doesn't look like these are used outside of `construct` so you could reduce scope here